content/notices/2018-10-24-navcoin-core-4-4-0.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: notices
+title: NavCoin Core 4.4.0
+author: Alex Vasquez
```

Comment:
s/vasquez/vazquez/

Reply:
😱my bad. btw did you know github has a new suggestion feature? I'll use it here so you can see